src/main/resources/graphql/schema.graphqls

```diff
@@ -9,33 +9,65 @@ type Article{
     publishedAt:String,
     scheduledAt:String,
     createdAt:String,
-    updatedAt:String
+    updatedAt:String,
+    unit:Unit
 }
 
-input ArticleInput{
+type ArticleRevision{
+    id:ID!,
     title:String,
     subTitle:String,
+    slug:String,
     body:String,
+    status:String,
+    tags:String,
```

Comment:
Article Revision has tags but tags field is missing in Article query, is this intentional?

Reply:
not at all 👎 i will look at it 👍